Sources/Fluent/SQL/GeneralSQLSerializer.swift

```diff
@@ -568,10 +568,14 @@ open class GeneralSQLSerializer<E: Entity>: SQLSerializer {
                 }
             }
         case .subset(let key, let s, let subValues):
-            statement += escape(filter.entity.entity) + "." + escape(key)
-            statement += scope(s)
-            statement += placeholders(subValues)
-            values += subValues
+            if subValues.count == 0 {
+                statement += "false"
```

Comment:
Hardcoding `false` here is okay for the `.in` scope, but will still yield invalid results for `.notIn`. (Considering set theory, it should return all records for an empty array.)